azure-pipelines/main.yml

```diff
@@ -1,23 +1,23 @@
-# Changes in master
```

Comment:
-R,  --repo <[HOST/]OWNER/REPO>
Select another repository using the [HOST/]OWNER/REPO format